libsolidity/ast/Types.cpp

```diff
@@ -3071,10 +3072,7 @@ u256 FunctionType::storageSize() const
 
 bool FunctionType::leftAligned() const
 {
-	if (m_kind == Kind::External)
-		return true;
-	else
-		solAssert(false, "Alignment property of non-exportable function type requested.");
+	return m_kind == Kind::External;
```

Comment:
Why was the assertion relaxed?

Reply:
In order to avoid the check whether or not the type is an external function type at the call site.